src/sidebar/util/test/time-test.js

```diff
@@ -49,67 +34,132 @@ describe('sidebar.util.time', function() {
     // test expectations
     const offset = new Date().getTimezoneOffset();
     if (offset > 0) {
-      sandbox.clock.tick(offset * 60 * 1000);
+      sandbox.clock.tick(offset * 60 * msPerSecond);
     }
+    fakeIntl = sinon.stub().returns({
```

Comment:
`Intl` is a namespace rather than a function. In the browser you access it via `Intl.DateTimeFormat` rather than say `Intl().DateTimeFormat` or `new Intl().DateTimeFormat`.